packages/sx05re/emuelec/bin/emuelecRunEmu.sh

```diff
@@ -385,6 +385,12 @@ case ${PLATFORM} in
             mame.sh
                 fi
                 ;;
+"psp")
+		if [ "$EMU" = "ppsspp_libretro" ]; then
+            set_kill_keys "ppsspp_libretro"
+            ppsspplib.sh
```

Comment:
seems redundant to check for those paths, program should already handle those directly, no?